eventAuditor: EventAuditor field was renamed on #256 merge

**`event-auditor` branch**

https://github.com/kubearmor/KubeArmor/commit/897cdcf9e0ec822674a2e5c608234800006126f5 renamed LogFeeder to `Logger`.

```bash
vagrant@kubearmor-dev-next:~/KubeArmor/KubeArmor/eventAuditor/BPF$ make tests V=1
INFO: compiling test /home/vagrant/KubeArmor/KubeArmor/eventAuditor/tests/test
CGO_LDFLAGS=/home/vagrant/KubeArmor/KubeArmor/eventAuditor/BPF/include/libbpf.a CGO_CFLAGS="-I /home/vagrant/KubeArmor/KubeArmor/eventAuditor/BPF/include" \
	go build -o /home/vagrant/KubeArmor/KubeArmor/eventAuditor/tests/test /home/vagrant/KubeArmor/KubeArmor/eventAuditor/tests/test.go
# github.com/kubearmor/KubeArmor/KubeArmor/eventAuditor
../sharedMapManager.go:74:6: ea.LogFeeder undefined (type *EventAuditor has no field or method LogFeeder)
../sharedMapManager.go:80:6: ea.LogFeeder undefined (type *EventAuditor has no field or method LogFeeder)
../sharedMapManager.go:86:6: ea.LogFeeder undefined (type *EventAuditor has no field or method LogFeeder)
../sharedMapManager.go:92:6: ea.LogFeeder undefined (type *EventAuditor has no field or method LogFeeder)
../sharedMapManager.go:98:6: ea.LogFeeder undefined (type *EventAuditor has no field or method LogFeeder)
../sharedMapManager.go:127:6: ea.LogFeeder undefined (type *EventAuditor has no field or method LogFeeder)
../sharedMapManager.go:134:6: ea.LogFeeder undefined (type *EventAuditor has no field or method LogFeeder)
make: *** [Makefile:116: /home/vagrant/KubeArmor/KubeArmor/eventAuditor/tests/test] Error 2
```

I'm fixing it.

--- a/KubeArmor/eventAuditor/sharedMapManager.go
+++ b/KubeArmor/eventAuditor/sharedMapManager.go
@@ -71,31 +71,31 @@ func (ea *EventAuditor) InitSharedMaps() error {
 
 		_, err = os.Stat(mapObjFilePath)
 		if errors.Is(err, os.ErrNotExist) {
-			ea.LogFeeder.Err(err.Error())
+			ea.Logger.Err(err.Error())
 			continue
 		}
 
 		bpfObj, err = lbpf.OpenObjectFromFile(mapObjFilePath)
 		if err != nil {
-			ea.LogFeeder.Err(err.Error())
+			ea.Logger.Err(err.Error())
 			continue
 		}
 
 		err = bpfObj.Load()
 		if err != nil {
-			ea.LogFeeder.Err(err.Error())
+			ea.Logger.Err(err.Error())
 			bpfObj.Close()
 		}
 
 		bpfMap, err = bpfObj.FindMapByName(mapName)
 		if err != nil {
-			ea.LogFeeder.Err(err.Error())
+			ea.Logger.Err(err.Error())
 			bpfObj.Close()
 		}
 
 		err = pinMap(bpfMap)
 		if err != nil {
-			ea.LogFeeder.Err(err.Error())
+			ea.Logger.Err(err.Error())
 		}
 
 		sharedMaps[mapName] = bpfMap
@@ -124,14 +124,14 @@ func (ea *EventAuditor) StopSharedMaps() error {
 
 		if bpfMap, found = sharedMaps[mapName]; !found {
 			errOnStopping[mapName]++
-			ea.LogFeeder.Errf("Map %s is not initialized to be stopped", mapName)
+			ea.Logger.Errf("Map %s is not initialized to be stopped", mapName)
 			continue
 		}
 
 		err = unpinMap(bpfMap)
 		if err != nil {
 			errOnStopping[mapName]++
-			ea.LogFeeder.Err(err.Error())
+			ea.Logger.Err(err.Error())
 		}
 
 		bpfMap.Object().Close()

--- a/KubeArmor/eventAuditor/tests/test.go
+++ b/KubeArmor/eventAuditor/tests/test.go
@@ -27,7 +27,7 @@ func main() {
 	var err error
 	var eAuditor ea.EventAuditor
 
-	eAuditor.LogFeeder = fd.NewFeeder("", "1337", "stdout", "", false)
+	eAuditor.Logger = fd.NewFeeder("", "1337", "stdout", "", false)
 
 	// First set the right path to load the ebpf object files
 	eAuditor.SetBPFObjPath("../" + ea.BPFObjRelPath)

